base/applications/rapps/settingsdlg.cpp

```diff
@@ -69,15 +69,15 @@ namespace
 
         CheckRadioButton(hDlg, IDC_PROXY_DEFAULT, IDC_USE_PROXY, IDC_PROXY_DEFAULT + Info->Proxy);
 
-		if (IDC_PROXY_DEFAULT + Info->Proxy == IDC_USE_PROXY)
+        if (IDC_PROXY_DEFAULT + Info->Proxy == IDC_USE_PROXY)
```

Comment:
I'm not a big fan about this, I think this would be more readable.
```suggestion
        if (Info->Proxy == 0)
```
And as a bonus suggestion, you can define an enum to get rid of those magic consts :P

Reply:
Also, instead of creating a commit to fix indentation, you'd better
```
git add <your files>
git commit --amend
```
to merge your new modifications into the existing commit (and then force-push)

Reply:
ok

Reply:
I can't add you change (marked "outdated"). Is it intentionnal ?

Reply:
The last commit didn't have the line where I wanted to suggest changes (hence my suggestion to amend your commit instead creating a new one for indentation fixes)

Reply:
ok